Jenkinsfile

```diff
@@ -283,11 +318,56 @@ pipeline {
             }
         }
 
+        stage('QA') {
+            when {
+                expression { params.RUN_QA }
+            }
+            environment {
+                IMAGE = "gcr.io/zeebe-io/zeebe"
+                VERSION = readMavenPom(file: 'parent/pom.xml').getVersion()
+                TAG = "${env.VERSION}-${env.GIT_COMMIT}"
+                DOCKER_GCR = credentials("zeebe-gcr-serviceaccount-json")
+                ZEEBE_AUTHORIZATION_SERVER_URL = 'https://login.cloud.ultrawombat.com/oauth/token'
+                ZEEBE_CLIENT_ID = 'W5a4JUc3I1NIetNnodo3YTvdsRIFb12w'
+                QA_RUN_VARIABLES = "{\"zeebeImage\": \"${env.IMAGE}:${env.TAG}\", \"generationTemplate\": \"${params.GENERATION_TEMPLATE}\", \"channel\": \"Internal Dev\", " +
+                                   "\"branch\": \"${env.BRANCH_NAME}\", \"build\": \"${currentBuild.absoluteUrl}\"}"
+            }
+
+            steps {
+                timeout(time: shortTimeout, unit: 'MINUTES') {
+                    container('docker') {
+                        sh 'cp dist/target/zeebe-distribution-*.tar.gz zeebe-distribution.tar.gz'
```

Comment:
The image is already available, so you could just take the existing `camunda/zeebe:current-test` and tag it differently, no need to rebuild it.

You can it like this:

```
docker tag camunda/zeebe:current-test ${IMAGE}:${TAG}
docker push ${IMAGE}:${TAG}
```